src/websocketserver.js

```diff
@@ -31,18 +39,22 @@ class WebSocketServer {
                 conn.drop(0, "Receiving messages not supported");
             });
 
-            conn.on("close", (code, desc) => {
-                console.log(`Closing connection: code: ${code} desc: ${desc}`);
+            conn.on("close", () => {
+                // console.log(`Closing connection: code: ${code} desc: ${desc}`);
```

Comment:
Er det meningen der skal være et udkommenteret console.log?

Reply:
Jest er ikke glad for det, men jeg vil gerne have at vi kigger på at reenable dette hvis det er muligt